src/views/AddPriceSingle.vue

```diff
@@ -169,12 +173,15 @@
                   @click:clear="clearProof"
                   :loading="createProofLoading">
                 </v-file-input>
-                <p v-if="proofFormFilled && !createProofLoading" class="text-green mt-2 mb-2">
+                <p v-if="proofFormFilled && !createProofLoading && !proofSelectedMessage" class="text-green mt-2 mb-2">
                   <i>{{ $t('AddPriceSingle.PriceDetails.ProofUploaded') }}</i>
                 </p>
                 <p v-if="!proofFormFilled && !createProofLoading" class="text-red mt-2 mb-2">
                   <i>{{ $t('AddPriceSingle.PriceDetails.UploadProof') }}</i>
                 </p>
+                <p v-if="proofFormFilled && proofSelectedMessage" class="text-green mt-2 mb-2">
```

Comment:
rather move this in the first `<p>` and add a v-if inside (avoids having 3 `<p>`)

Reply:
Since there is already a v-if in the first `<p>` element, I don't think that would simplify. What I could do is group the 3 `<p>` together with a computed method

Reply:
currently
![Screenshot from 2024-02-25 23-40-24](https://github.com/openfoodfacts/open-prices-frontend/assets/7147385/6224c8a1-8301-407d-ab98-58e0411604af)

what I meant
![Screenshot from 2024-02-25 23-41-32](https://github.com/openfoodfacts/open-prices-frontend/assets/7147385/1cab5432-fb00-453d-bf8f-04dbd4212dd7)

which one is simpler ? ^^

Reply:
Alright, though I still think one <p> and a computed method is better :) But that can come in the future